java/src/main/native/src/NvtxRangeJni.cpp

```diff
@@ -25,15 +25,17 @@ Java_ai_rapids_cudf_NvtxRange_push(JNIEnv *env, jclass clazz,
     jstring name, jint color_bits) {
   try {
     cudf::jni::native_jstring range_name(env, name);
-    cudf::nvtx::range_push_hex(range_name.get(), color_bits);
+    nvtx3::color range_color(static_cast<nvtx3::color::value_type>(color_bits));
+    nvtx3::event_attributes attr{range_color, range_name.get()};
+    nvtxDomainRangePushEx(nvtx3::domain::get<nvtx3::domain::global>(), attr.get());
```

Comment:
I'd suggest creating a "Java" or "Spark" domain. Nsight will separate out NVTX calls that come from different domains.

It's pretty easy to do this:

```
struct java_domain{
   static constexpr char const* name{"Java"};
};
...
nvtxDomainRangePushEx(nvtx3::domain::get<java_domain>(), attr.get());
```

Reply:
cudf's Java bindings are not supposed to be specific to Spark, so I went with a "Java" domain for now.  We can extend the Java NVTX range API in the future to support app-specified domains if desired.